buildkite/src/Command/OpamInit.dhall

```diff
@@ -23,6 +26,16 @@ let commands : List Text -> List Cmd.Type = \(environment : List Text) ->
       Cmd.CacheSetupCmd::{
         create = r "make setup-opam",
         package = r "tar cfz ${file} /home/opam/.opam"
+      },
+    Cmd.cacheThrough
+      Cmd.Docker::{
+        image = (../Constants/ContainerImages.dhall).minaToolchain,
+        extraEnv = environment
+      }
+      local
+      Cmd.CacheSetupCmd::{
+        create = r "[ -d _opam ] && : || make setup-opam",
+        package = r "tar cfz ${local} _opam"
```

Comment:
what's happening in this file exactly :o ?

Reply:
When setting up opam, both the system and local switches need to be cached on the CI.

I've modified the file to ensure that the local opam switch is generated, (un)tarballed, and cached on the CI as necessary.